indispensable/src/commonMain/kotlin/at/asitplus/signum/indispensable/pki/X509Certificate.kt

```diff
@@ -320,9 +393,19 @@ private
 /** De-/serializes Base64 strings to/from [ByteArray] */
 object Asn1BitStringSerializer : TransformingSerializerTemplate<Asn1BitString?, String>(
     parent = String.serializer(),
-    encodeAs = { if (it == null) "" else byteArrayOf(it.numPaddingBits, *it.rawBytes).encodeToString(Base64Strict) },
+    encodeAs = {
+        if (it == null) "" else byteArrayOf(
+            it.numPaddingBits,
+            *it.rawBytes
+        ).encodeToString(Base64Strict)
+    },
```

Comment:
please don't add unnecessary whitespace changes to a pull request, they just bloat the review diff